File: big-O/finding_nemo.py
import time
nemo = ['demo', 'gemo', 'cemo']

def finding_nemo (array):
    start_time = time.time()
    for i in array:
        if i == 'nemo':
            print("Found NEMO!")
            return

    print("Nemo is not here")
    print("---%s seconds ---"% (time.time()-start_time))


finding_nemo(nemo)

File: big-O/finding_nemo.js
const performance = require('perf_hooks').performance;
const nemo = ['demo', 'gemo', 'cemo'];

function finding_nemo (array) {

  for (var i=0; i < array.length; i++) {
    var t0 = performance.now();
    if (array[i] === 'nemo') {
      console.log("Found NEMO!");
      return
    }

  }
  console.log("Nemo is not here");
  var t1 = performance.now();
  console.log("Runtime is "+(t1 - t0)+" miliseconds");
}

finding_nemo(nemo);
